config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: 506463145083.dkr.ecr.us-west-2.amazonaws.com/cwagent-operator-pre-release
-  newTag: latest
+  newName: public.ecr.aws/cloudwatch-agent/cloudwatch-agent-operator
+  newTag: 1.0.2
```

Comment:
Is this being used for `make deploy` as well? 
Did we actually push to our public repo while running these github tests? Id imagine it atleast complains about permission issues. Mind checking?

Reply:
We shouldnt be using our public repo and tags here.

Reply:
No we are not using this in make deploy. I have updated it now.